### PR TITLE
feat(getTags): send query params object

### PR DESCRIPTION
**Pull request by phoebeschmidt**
We should send a query params object in getTags (for consistency with our other get-collection endpoints in the SDK).

### CODE REVIEW

File: lib/create-environment-api.ts [hunk 1]
```diff
@@ -1023,9 +1023,9 @@ export default function createEnvironmentApi({
         })
         .then((response) => wrapTag(http, response.data), errorHandler)
     },
-    getTags(skip?: number, limit?: number) {
+    getTags(query: QueryOptions = {}) {
```
Comment (TimBeyer) [inline lib/create-environment-api.ts:1026]: The type isn't entirely applicable here.
It includes `content_type` and `include`.

See https://github.com/contentful/contentful-management.js/blob/28483c2cb8c36f874837a7de15724c9ab7d6315f/lib/common-types.ts#L46-L54

Probably we should have a separate type with all the allowed params.

**Comment by TimBeyer** [comment]
This is a breaking change so probably it should be a `feat!`

File: lib/create-environment-api.ts (hunk 1 above)
Comment (phoebeschmidt) [reply lib/create-environment-api.ts:1026]: great catch! yep will do

**Comment by phoebeschmidt** [comment]
@TimBeyer if it should be a new major version then it should be a `perf` ([relevant sem-rel docs](https://github.com/semantic-release/semantic-release#commit-message-format)). But this shouldn't actually break for anybody, since no one is using it. 

Feat seems right to me. WDYT?

**Pull request merged**